roles/kubernetes-apps/ansible/templates/nodelocaldns-daemonset.yml.j2

```diff
@@ -47,17 +47,18 @@ spec:
         - coredns
 {% if enable_nodelocaldns_secondary %}
         - -skipteardown
-{% else %}
+{% endif %}
         ports:
+        - containerPort: 9253
```

Comment:
Unsure of why this is isn't the case, but maybe we should use `nodelocaldns_prometheus_port` here, for consistency with the prom annotations scrape method.

Not sure why we don't have a variable defined for the secondary metrics port though.

Reply:
There is a `nodelocaldns_secondary_prometheus_port` which I've updated the PR to use. PTAL.